### PR TITLE
Update lab-07-1-learning_rate_and_evaluation.py

### PR DESCRIPTION
**Pull request by qoocrab**
1. Modified to a learning rate that is appropriate for learning.
2. The arg_max is modified to argmax.

### CODE REVIEW

**Comment by kkweon** [comment]
Thank you!
